### PR TITLE
Dialog boxes

### PR DESCRIPTION
**Pull request by b-w-cole**

### CODE REVIEW

**Comment by b-w-cole** [comment]
The commits look a little wonky - really, this PR only contains the Dialog Box test.  
I mistakenly branched off of the Lists and Dropdowns test.

File: tests/dialog-boxes.spec.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    await expect(page.locator('label')).toHaveText('Name')
+    
+    // Validate that the input field is visible
+    const input = page.locator('#name')
+    await expect(input).toBeVisible()
+
+    /* 5. Add a new pet type with the name "pig" and click "Save" button */
+    // Adding 'pig' as the new pet type
+    await input.click()
+    await page.keyboard.type('pig')
```
Comment (ba-instructor) [inline tests/dialog-boxes.spec.ts]: Keep it simple. Not necessary to simulate the keyboard keystrokes. It was not a requirement.
Lines 35 and 36 can be just a single line: `await input.fill('pig')`

**Review by ba-instructor — APPROVED** [review]
All right, all looks good!
Just a single comment.

Good job!

File: tests/dialog-boxes.spec.ts (hunk 1 above)
Comment (b-w-cole) [reply tests/dialog-boxes.spec.ts]: Thanks! I will correct this and then merge.

**Pull request merged**